docs/guide/quick-start.md

```diff
@@ -73,9 +72,8 @@ class AuthHandler
         if (Yii::$app->user->isGuest) {
             if ($auth) { // login
                 /* @var User $user */
-                $user = $auth->user;
-                $this->updateUserInfo($user);
```

Comment:
Removing this you won't get updates for the data from the authentication provider. For example, if I'll set "nickname" at Facebook while it being empty before, it won't be reflected in the application. That isn't always a desired behavior but it it quite common.

Reply:
Thanks for response. I understand about your opinion about nickname. But, there is another thing that need to clarify. That is about github column in user table. Would it better if we add github column in [this migration script](https://www.yiiframework.com/extension/yiisoft/yii2-authclient/doc/guide/2.2/en/installation#storing-authorization-data) ?

Reply:
Yeah. That's a good idea.